simple_scripts/faraday_start.py

```diff
@@ -0,0 +1,76 @@
+from subprocess import call
+import os
+import time
+import sys
+
+#Detect operating system
+#LINUX: Assumed terminal is gnome-terminal standard with Ubuntu
+running_os = sys.platform
+
+# Proxy
+proxy_unitcnt = 1  # Number of units proxy is connecting to (starts at unit0)
+proxy_unit0_callsign = 'REPLACEME'
+proxy_unit0_nodeid = 0
```

Comment:
Yep just like your prior comment. Updating with a comment "Replace with proxy_nodeid of local unit to be programmed"